projects/demo/src/modules/pipes/is-present/examples/1/component.ts

```diff
@@ -0,0 +1,36 @@
+import {Component} from '@angular/core';
+import {FormControl} from '@angular/forms';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {tuiIsPresent} from '@taiga-ui/cdk';
+
+@Component({
+    selector: `tui-is-present-example1`,
+    templateUrl: `./template.html`,
+    styleUrls: [`./style.less`],
+    changeDetection,
+    encapsulation,
+})
+export class TuiIsPresentExample1 {
```

Comment:
not sure I understand this example

Reply:
it's basically a copy-paste from the existing [`tuiIsPresent`](https://github.com/Tinkoff/taiga-ui/blob/main/projects/demo/src/modules/utils/miscellaneous/examples/5/index.ts) function example

Reply:
but why should we show how the pipe works in this way?

instead

```html
{{ value | tuiPresent }} = value | tuiPresent
```

Reply:
oh, I see. you're right

Reply:
updated